### ISSUE TITLE
Typing should autofocus the chatbar

### ISSUE THREAD

**Issue by Freeplayg** [oxen-io/session-desktop#2507]
Feels a lot nicer to not have to click into the chatbar every time, feels finicky.

**Comment by KeeJef** [comment]
If we start using more keyboard shortcuts we should make sure this doesn't interfere

**Comment by Bilb** [comment]
yeah we have a few shortcut already set up (page up and down scroll in the conversation list when it is selected). 
But making the composition box autoselected when typing something else that the shortcuts should be easy.
